service-front/app/src/Actor/src/Form/CreateNewActivationKey.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Actor\Form;
+
+use Common\Form\AbstractForm;
+use Mezzio\Csrf\CsrfGuardInterface;
+use Laminas\InputFilter\InputFilterProviderInterface;
+
+/**
+ * Class CreateNewActivationKey
+ * @package Actor\Form
+ */
+class CreateNewActivationKey extends AbstractForm implements InputFilterProviderInterface
+{
+    const FORM_NAME = 'create_new_activation_key';
```

Comment:
```suggestion
    public const FORM_NAME = 'create_new_activation_key';
```
This will fix the linting error below